### PR TITLE
#109 Implement UDIM style syntax for padding

### PR DESCRIPTION
**Pull request by herronelou**
Implementation of UDIM style padding syntax.

### CODE REVIEW

File: src/fileseq/filesequence.py [hunk 1]
```diff
@@ -1296,6 +1296,9 @@ def getPaddingNum(cls, chars, pad_style=PAD_STYLE_DEFAULT):
         if not chars:
             return 0
 
+        if chars in ['<UDIM>', '%(UDIM)d']:
```
Comment (herronelou) [inline src/fileseq/filesequence.py:1299]: Looking back at this, maybe this list should be in the constants?

File: src/fileseq/constants.py [hunk 2]
```diff
@@ -65,7 +65,8 @@ def __eq__(self, other):
 # Example: /film/shot/renders/bilbo_bty.1-100@.exr
 # Example: /film/shot/renders/bilbo_bty.1-100@@@@#.exr
 # Example: /film/shot/renders/bilbo_bty.1-100%04d.exr
-# Example: /film/shot/renders/bilbo_bty.1-100$F4d.exr
+# Example: /film/shot/renders/bilbo_bty.1-100$F4.exr
```
Comment (justinfx) [inline src/fileseq/constants.py:68]: Thanks for catching this typo

File: src/fileseq/filesequence.py (hunk 1 above)
Comment (justinfx) [reply src/fileseq/filesequence.py:1299]: It wouldn't hurt to keep the values all in one place.

**Review by justinfx — APPROVED** [review]
Thanks for the contribution! Looks fine to me.

**Review by justinfx — APPROVED** [review]

**Comment by justinfx** [comment]
Thanks!

**Pull request merged**